libraries/provider_service.rb

```diff
@@ -0,0 +1,89 @@
+# Chef Provider for configuring an kibana service in the init system
+class KibanaCookbook::ServiceProvider < Chef::Provider::LWRPBase
+  provides :kibana_service
+  include KibanaCookbook::Helpers
+
+  def whyrun_supported?
+    false
+  end
+
+  action :remove do
+    raise "#{new_resource} remove not currently implemented"
+  end
+
+  action :configure do
+    kb_install = find_kb_resource(run_context, :kibana_install, new_resource)
+    kb_user = find_kb_resource(run_context, :kibana_user, new_resource)
+    # Create service
+    #
+    if kb_install.version == 'tarball'
+      init_r = template "/etc/init/#{new_resource.service_name}.conf" do
+        source new_resource.init_source
+        cookbook new_resource.init_cookbook
+        owner 'root'
+        mode 0o0755
```

Comment:
it's right. It's required by rubocop or you can use the string but IMHO use octets better.